### PR TITLE
[Tests-Only] Fetch API testrunner from specific branch and commit

### PR DESCRIPTION
**Pull request by phil-davis**
Part of issue https://github.com/owncloud/ocis/issues/303

Specify the branch, and commit in that branch, to use when fetching the ownCloud10 core API test runner code.

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -100,19 +100,20 @@ def apiTests(ctx):
...
-              'name': 'gopath',
-              'path': '/srv/app',
-            },
-          ]
+        },
+        'commands': [
+          'git clone -b master --depth=1 https://github.com/owncloud/testing.git /srv/app/tmp/testing',
+          'git clone -b %s --single-branch --no-tags https://github.com/owncloud/core.git /srv/app/testrunner' % (coreBranch),
+          'cd /srv/app/testrunner',
+          'git checkout %s' % (coreCommit),
```
Comment (dpakach) [inline .drone.star:108]: can we make it to ignore the commit hash when it is not given, for eg when you want to run just on the latest commit of some branch.

Comment (phil-davis) [reply .drone.star:108]: You can specify the branch name again, and it will `git checkout master`

Is that good enough?

Comment (dpakach) [reply .drone.star:108]: or we can just add something like
```
if (coreCommit == None):
    coreCommit = coreBranch
```
it would look it better IMHO

Comment (phil-davis) [reply .drone.star:108]: I will fix it properly - it is easy!

**Pull request merged**